frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -537,7 +537,7 @@ const CSRTableRow: React.FC<RowProps<NodeCertificateSigningRequestKind>> = ({
         id={nodeColumnInfo.created.id}
         activeColumnIDs={activeColumnIDs}
       >
-        <Timestamp timestamp={csr.metadata.creationTimestamp} />
+        <Timestamp timestamp={csr.metadata?.creationTimestamp || ''} />
```

Comment:
should no longer be needed as Timestamp now accepts null

```suggestion
        <Timestamp timestamp={csr.metadata?.creationTimestamp} />
```